Engage Cockpit CM/CD Documentation + Settings Section (#7790)
umbraco/UmbracoDocs: 17/umbraco-engage/getting-started/for-marketers-and-editors/cockpit.md

If the Cockpit is missing while the Umbraco backoffice runs on a different domain, contact the technical team. You can also refer to the [Load Balancing and CM/CD Environments](../for-developers/loadbalancing-and-cm-cd-environments.md) article.

### Opening the Cockpit from the Backoffice

> If your website runs on a different server than the Umbraco backoffice (a CM/CD setup), you may not see the Cockpit automatically when browsing the frontend.

Reviewer: Another suggestion to shorten a sentence.

```suggestion
If your website runs separately from the backoffice (a CM/CD setup), you may not see the Cockpit automatically when browsing the frontend.
```
